client/js/libs/handlebars/tz.js

```diff
@@ -2,6 +2,6 @@
 
 const moment = require("moment");
 
-module.exports = function(time) {
-	return moment(time).format("HH:mm");
+module.exports = function(time, seconds) {
```

Comment:
There's something a little bit weird about passing through seconds here, but I don't know what the better option would be, tbh. If we had control over requring these handlebars templates, then we could use the modules that we are creating, but webpack pulls them in itself, so we'll end up with differing settings, which isn't good. So, I guess this is just me moaning about the system rather than asking you to do anything.